types/tough-cookie-filestore/index.d.ts

```diff
@@ -0,0 +1,47 @@
+// Type definitions for tough-cookie-filestore 0.0
+// Project: https://github.com/mitsuru/tough-cookie-filestore
+// Definitions by: Christian Friedow <https://github.com/friedow>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = FileCookieStore;
+export as namespace FileCookieStore;
```

Comment:
This line should be removed; as far as I can tell, `tough-cookie-filestore` is not a UMD module.

Reply:
That is, `export as namespace` is only relevant when the underlying module is a UMD module.

Reply:
Changed that, thanks for the explanation 👍 .